src/constants.ts

```diff
@@ -21,34 +20,30 @@
  * @property {'Uncategorized'} Uncategorized - for any changes that have not
  *   yet been categorized.
  */
-
-/**
- * @type {ChangeCategories}
- */
-const changeCategories = {
-  Added: 'Added',
-  Changed: 'Changed',
-  Deprecated: 'Deprecated',
-  Fixed: 'Fixed',
-  Removed: 'Removed',
-  Security: 'Security',
-  Uncategorized: 'Uncategorized',
-};
+export enum changeCategories {
```

Comment:
Could we make this singular instead? I found this rather confusing.

Previously `changeCategories` made sense because it was a collection. But now that it's an enum, it's not really a collection anymore. Any use of this enum does refer to a _single_ category.

For example: ```type ReleaseChanges = Record<changeCategories, string[]>;```
Here we're mapping a _single category_ to an array of strings, not a collection of categories to an array of strings.

Reply:
Done in: 8323b60